apps/cookbook/src/app/examples/fab-sheet-example/fab-sheet-example.component.html

```diff
@@ -1,10 +1,10 @@
 <kirby-fab-sheet horizontalAlignment="right" [disabled]="disableFabSheet">
   <kirby-icon name="write-message"></kirby-icon>
-  <kirby-action-sheet
+  <kirby-action-sheet-popout
```

Comment:
Changed the API to avoid breaking change. It is now possible to optionally instruct the action-sheet to hide the trigger button. FAB and page showcases was updated to reflect this